win32ss/user/ntuser/sysparams.c

```diff
@@ -1404,10 +1404,10 @@ SpiGetSet(UINT uiAction, UINT uiParam, PVOID pvParam, FLONG fl)
             return SpiGetInt(pvParam, &gspv.bFontSmoothing, fl);
 
         case SPI_SETFONTSMOOTHING:
-            gspv.bFontSmoothing = (uiParam == 2);
+            gspv.bFontSmoothing = uiParam;
```

Comment:
You can use this to convert it into a BOOL:
```suggestion
            gspv.bFontSmoothing = !!uiParam;
```

Reply:
I think I would rather do this if we want to be more explicit, though I don't think its necessary:
`gspv.bFontSmoothing = uiParam ? TRUE : FALSE`
I'll leave this up for debate.